packages/tools/src/stateManagement/segmentation/config/segmentationVisibility.ts

```diff
@@ -115,9 +121,11 @@ function getSegmentationVisibility(
     return;
   }
 
-  const { segmentsHidden } = representation;
+  const { segmentsHidden, segmentationId } = representation;
+  const indices = getSegmentationIndices(segmentationId);
```

Comment:
I'm not a fan of this function name (which I think I came up with), but could we consider renaming it to `const segmentIndices = getUniqueSegmentIndices`?